src/osd/osd_types.h

```diff
@@ -3986,7 +3986,7 @@ struct object_info_t {
 
   flag_t flags;
 
-  static string get_flag_string(flag_t flags) {
+  string get_flag_string(flag_t flags) const {
```

Comment:
get_flag_string will not change the members inside class, so it should be constness. When I modify it, I think "static" could also be removed, since the functions with similar usage don't use static flag.